CHANGELOG.md

```diff
@@ -3,8 +3,17 @@
 The following is a curated list of changes in the Enact agate module, newest changes on the top.
 
 ## [unreleased]
+
+### Added
+
+- `agate/Button` `checkPropsIncompatibility` function to warn props incompatibility
```

Comment:
`checkPropsIncompatibility` is not `agate/Button/`'s prop. It might be better to remove it.